frontend/src/NavBar.tsx

```diff
@@ -0,0 +1,31 @@
+import {useNavigate} from "react-router-dom";
+import Button from "./css/Button";
+import React from "react";
+
+export default function NavBar() {
+    const navigate = useNavigate()
+
+    const routeToLoginAndRegiter = () => {
```

Comment:
Why do you need `routeToLoginAndRegiter` and `routeToLogin` when they both do the same.